.circleci/config.yml

```diff
@@ -64,11 +64,13 @@ jobs:
           destination: libNFHTTP.zip
   buildmacandroid:
     macos:
-      xcode: "9.0"
+      xcode: "11.5.0"
     steps:
       - checkout
       - run: git submodule sync
       - run: git submodule update --init --recursive
+      # Android NDK does not pass.
+      - run: sudo spctl --master-disable
```

Comment:
@drubinstein for the future in case you run into anything similar: this build was timing out because the android ndk was causing a 10.15 (Catalina) permissions prompt since the android ndk binaries are unverified. This disables that checking system-wide.